src/test/java/com/targomo/client/api/request/TimeVectorRequestTest.java

```diff
@@ -0,0 +1,125 @@
+package com.targomo.client.api.request;
+
+import com.targomo.client.api.TravelOptions;
+import com.targomo.client.api.enums.EdgeWeightType;
+import com.targomo.client.api.enums.TravelType;
+import com.targomo.client.api.exception.TargomoClientException;
+import com.targomo.client.api.geo.DefaultSourceCoordinate;
+import com.targomo.client.api.response.TimeVectorResponse;
+import com.targomo.client.api.response.parsingpojos.TransitTravelTimes;
+import org.apache.commons.io.IOUtils;
+import org.glassfish.jersey.message.GZipEncoder;
+import org.junit.Test;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TimeVectorRequestTest extends RequestTest {
+
+    private static final String INTEGRATION_TEST_KEY = "T17ZPR4F190APX61I1DR141142829";
```

Comment:
Is it ok to have this key here? Just asking.

Reply:
Because in other places of the code we can see the message "YOUR API KEY HERE" instead

Reply:
it's a "free" test key - so nothing any people cannot already do with our API